addon/templates/components/frost-select-dropdown.hbs

```diff
@@ -10,12 +10,12 @@
 />
 <div
   class='frost-select-dropdown'
-  data-test={{hook (concat receivedHook '-list')}}
+  data-test={{hook (concat hook '-list')}}
```

Comment:
I think we actually want to use `hookPrefix` here but nothing in core currently does that. Given that is the case I figured a separate PR can convert everything to use `hookPrefix` instead of `hook` in the templates given it doesn't cause any breaking changes.